Source/Blazorise/Components/Validation/ValidationRule.cs

```diff
@@ -182,5 +184,29 @@ public static void IsFileSelected( ValidatorEventArgs e )
         }
     }
 
+    /// <summary>
+    /// Checks if the date is selected.
+    /// </summary>
+    /// <typeparam name="TValue">Data-type used be binded by the Date property.</typeparam>
+    /// <param name="e"></param>
+    public static void IsDateSelected<TValue>( ValidatorEventArgs e )
+    {
+        var dates = e.Value as TValue[];
+
+        e.Status = dates?.Count( x => !x.IsEqual( default ) ) >= 1 ? ValidationStatus.Success : ValidationStatus.Error;
+    }
+
+    /// <summary>
+    /// Checks if the date range is selected.
+    /// </summary>
+    /// <typeparam name="TValue">Data-type used be binded by the Dates property.</typeparam>
+    /// <param name="e"></param>
+    public static void AreDatesSelected<TValue>( ValidatorEventArgs e )
+    {
+        var dates = e.Value as IReadOnlyList<TValue>;
+
+        e.Status = dates?.Count( x => !x.IsEqual( default ) ) >= 2 ? ValidationStatus.Success : ValidationStatus.Error;
```

Comment:
Ah, I see why you used Count up above...
I have to say I don't exactly get what this is doing, it has a bunch of dates that represent date ranges, that it? So whenever there's a data range it's valid?

Could for example the count be 3 and still be valid?

Reply:
Any selection that is above or equal to 2 is valid. This is only a default behavior that is true for most use-cases. For any custom validation users would have to write their own logic for their business.